README.md

```diff
@@ -10,6 +10,14 @@ First, ensure that kubectl is installed and configured.
 
 ### Installation
 
+#### Quick Install (Linux & MacOS only)
+
+```shell
+curl -sSL https://raw.githubusercontent.com/GoogleCloudPlatform/kubectl-ai/main/install.sh | bash
+```
+
+#### Manual Installation
```

Comment:
can we add `(Linux, MacOS and Windows)` here. I know it is verbose, but do want users to think that windows is not supported.